src/Command/CreateGithubIssueFromMantisIssue.php

```diff
@@ -34,42 +35,69 @@ public function __construct(MantisConnector $mantisConnector, GithubConnector $g
 
     protected function configure()
     {
-        $this->setName('mantis2github');
+        $this->setName('sync');
```

Comment:
aber, aber, was mach ich denn jetzt mit dem Eintrag in meiner .zshrc: `alias m2g='cd /Users/sidler/web/mantis2github &&  php -f cli.php mantis2github'`? 😭

Reply:
Auf `alias m2g='cd /Users/sidler/web/mantis2github && php mantis2github sync'` aktualisieren 😄